### DISCUSSION TITLE
Rate limit Error for public.ecr.aws/awsguru/aws-lambda-adapter:0.8.2

### DISCUSSION THREAD

**Discussion by rclod** [awslabs/aws-lambda-web-adapter#438] [Q&A]
Getting this error when pulling from public.ecr.aws/awsguru/aws-lambda-adapter:0.8.2

Error: invalid from flag value public.ecr.aws/awsguru/aws-lambda-adapter:0.8.2: toomanyrequests: Rate exceeded

The error is not present when requesting a specific architecture such as public.ecr.aws/awsguru/aws-lambda-adapter:0.8.2-aarch64

**Comment by bnusunny** [comment]
ECR Public repo allows 1 request/second for un-authenticated pulls and 10 request/second for authenticated pulls. 

You can retry the operation a bit later or login to ECR Public and pull again. 

Here is the command to login ECR Public. 

```shell
aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws
```
